apps/sms/test/marionette/generators/thread.js

```diff
@@ -59,8 +59,8 @@ var ThreadGenerator = {
 
     for (var i = 0; i < parameters.numberOfMessages; i++) {
       var message = null;
-      var messageBody = thread.body + ' (messageId: ' +
-        (ThreadGenerator.uniqueMessageId + 1) + ')';
+      var messageBody = thread.body +
+        (i ? ' (messageId: ' + ThreadGenerator.uniqueMessageId + ')' : '');
```

Comment:
__question:__ what does this change mean? :) Looks like now ```messageId``` doesn't correspond to real message id?